Xamarin.Essentials/HapticFeedback/HapticFeedback.macos.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+using AppKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
+
+        static void PlatformPerform(HapticFeedbackType type)
+        {
+            if (type == HapticFeedbackType.LongPress)
+                NSHapticFeedbackManager.DefaultPerformer.PerformFeedback(NSHapticFeedbackPattern.Alignment, NSHapticFeedbackPerformanceTime.Default);
```

Comment:
Is alignment the correct pattern? What is that? How does it compare to generic?

Reply:
@mattleibow 
I didn't quite understand the question.
In total there are three types of the haptic feedback on MacOS:
- alignment
- Generic ?
- LevelChange ?
As I could tactually feel it, it seemed...

Reply:
That is what I mean. You picked alignment. Is that the correct one for long press? Why are the others not the right one?

Reply:
The reason I say this is because the docs say:
> A haptic feedback pattern to be used in response to the alignment of an object the user is dragging around.

Reply:
It may be replaced by "generic"